include/industry_standard/spdm.h

```diff
@@ -19,6 +19,7 @@
 #define SPDM_MAX_SLOT_COUNT 8
 #define SPDM_MAX_OPAQUE_DATA_SIZE 1024
 #define SPDM_MAX_CSR_TRACKING_TAG 7
+#define LIBSPDM_MAX_KEY_PAIR_COUNT 16
```

Comment:
you need to move this to spdm_lib_config.h

Reply:
Thanks. I have changed the location.